### PR TITLE
BE 7 - card (edit appointment)

### PR DESCRIPTION
**Pull request by KuzminAanton**

### CODE REVIEW

File: src/modules/controllers/appointment.controller.js [hunk 1]
```diff
@@ -1 +1,59 @@
 const Appointment = require('../../db/models/appointment/index');
+const Doctors = require('../../db/models/doctors/index');
+
+module.exports.addAppointments = async (req, res) => {
+  const {
+    patientName, doctorName, date, complaints,
+  } = req.body;
+  if (req.body) {
```
Comment (usatovaDasha) [inline src/modules/controllers/appointment.controller.js:8]: if a send empty data?

File: src/modules/controllers/appointment.controller.js [hunk 2]
```diff
@@ -1 +1,59 @@
...
+    });
+  });
+};
+
+module.exports.editAppointments = async (req, res) => {
+  const {
+    patientName, doctorName, date, complaints,
+  } = req.body;
+  const { _id } = req.query;
+  if (_id && patientName && doctorName && date && complaints) {
```
Comment (usatovaDasha) [inline src/modules/controllers/appointment.controller.js:48]: if i want change only date, or only doctor?

Comment (KuzminAanton) [reply src/modules/controllers/appointment.controller.js:48]: fixed

File: src/modules/controllers/appointment.controller.js [hunk 3]
```diff
@@ -45,7 +45,7 @@ module.exports.editAppointments = async (req, res) => {
     patientName, doctorName, date, complaints,
   } = req.body;
   const { _id } = req.query;
-  if (_id && patientName && doctorName && date && complaints) {
+  if (_id || patientName || doctorName || date || complaints) {
```
Comment (usatovaDasha) [inline src/modules/controllers/appointment.controller.js:48]: id optional parameter?

**Pull request merged**